swan-cern/values.yaml

```diff
@@ -351,8 +351,3 @@ gpu-operator:
     config:
       name: nvidia-device-plugin-config
       default: "default"
-  node-feature-discovery:
-    image:
```

Comment:
Do we still need to include this somewhere, though? In the scope of which chart does cern-magnum define it?

Reply:
In the scope of their own chart, here: https://gitlab.cern.ch/kubernetes/automation/releases/cern-magnum/-/blob/master/values.yaml?ref_type=heads#L608

Reply:
But this means we do not deploy anymore any component that needs these settings? There's no node-feature-discovery anymore?

Reply:
I believe it is deployed with the CERN magnum chart, at cluster creation time

Reply:
I see, and before it was deployed by the gpu operator chart? So after this update we won't have the node-feature-discovery pods anymore in our cluster until we move to the new template?

Reply:
It will still be deployed, but through cern-magnum, not the gpu-operator

Reply:
But the cern-magnum of our current cluster is the old cern-magnum which does not have this setting, right? That's why I'm asking if before the node-feature-discovery was deployed via the gpu operator chart.